get-started/part3.md

```diff
@@ -174,8 +200,7 @@ docker stack deploy -c docker-compose.yml getstartedlab
 Docker will do an in-place update, no need to tear the stack down first or kill
 any containers.
 
-Now, re-run the `docker stack ps` command to see the deployed instances reconfigured. For example, if you scaled up the replicas, there will be more
-running containers.
+Now, re-run `docker container ls -q` to see the deployed instances reconfigured. For example, if you scaled up the replicas, there will be more running containers.
```

Comment:
Maybe "more containers are started."

Reply:
Here it might be useful to make the distinction and say "more tasks are started, which means you will see more running containers"